changelog/24.0/24.0.0/summary.md

```diff
@@ -24,6 +25,22 @@ Previously, all window function queries required single-shard routing, which lim
 
 For examples and more details, see the [documentation](https://vitess.io/docs/24.0/reference/compatibility/mysql-compatibility/#window-functions).
 
+#### <a id="tablet-targeting"/>Tablet targeting via USE statement</a>
```

Comment:
@nickvanw let's say I use this feature to write rows to a `PRIMARY` that doesn't align with the VSchema, am I creating any data correctness/etc risks we should detail here?

I'm wondering if that would cause query failures or potentially-surprising results. But at the same time, I guess you can only use this feature with intention 🤔

And on a similar track, could vReplication get upset about rows in the wrong shard? One dreamed-up scenario I can think of is merging shards, and the same primary key exists in both, etc

None of what I'm thinking means changing the code here I think (it looks good so far), maybe just documentation